shynet/dashboard/templates/dashboard/pages/service.html

```diff
@@ -109,8 +109,24 @@
             <tbody>
                 {% for location in stats.locations %}
                 <tr>
-                    <td class="truncate w-full max-w-0">{{location.location|default:"Unknown"|urldisplay}}</td>
-                    <td class="rf">{{location.count|intcomma}}</td>
+                    <td class="truncate w-full max-w-0 relative">
+                        <div
+                            class="absolute h-6"
+                            style="width: {{location.count|percent:stats.hit_count}}; top: 6px; left: 6px; height: calc(100% - 12px); background-color: var(--color-urge-200-fallback)"
```

Comment:
`left: 0;` to align bar with the table header/body seperator line and make it not get cut off by the `td` tag when bar is full or close to full width